x-pack/plugins/security_solution/public/flyout/document_details/left/components/related_cases.tsx

```diff
@@ -30,13 +30,16 @@ const columns: Array<EuiBasicTableColumn<RelatedCase>> = [
         defaultMessage="Name"
       />
     ),
-    render: (value: string, caseData: RelatedCase) => (
-      <CellTooltipWrapper tooltip={caseData.title}>
-        <CaseDetailsLink detailName={caseData.id} title={caseData.title}>
-          {caseData.title}
-        </CaseDetailsLink>
-      </CellTooltipWrapper>
-    ),
+    render: (value: string, caseData: RelatedCase) => {
+      const index = data.findIndex((d) => d.id === caseData.id);
```

Comment:
In the existing guided onboarding tour's logic, it only shows the tour step if index equals to 0